benchmarks/distributed/pipeline/pipe.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
```

Comment:
Nit: Not sure whats the guidelines around these copyrights, but it seems like only the pipeline files have this, and no other files in OSS pytorch (i.e. caffe2/torch directory)

Reply:
I think we probably need it only for original torchgpipe files, will remove it for this.